fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -400,7 +400,7 @@ public Integer readResourceTypeId(String resourceType) throws FHIRPersistenceDBC
             resultSet = stmt.executeQuery();
             dbCallDuration = (System.nanoTime()-dbCallStartTime)/1e6;
             if (log.isLoggable(Level.FINE)) {
-                log.fine("DB search for ids complete. executionTime=" + dbCallDuration + "ms");
+                log.fine("DB search for ids complete. " + queryData + "  executionTime=" + dbCallDuration + "ms");
```

Comment:
I think a comment should be added as to what is logged and why, and also indicate that PHI could be logged here. 

Also should a `[` and `]` be used to delineate the queryData output? It might already be wrapped in the output of toString.

Reply:
yeah, queryData.toString is like `[queryString=" + queryString + ", bindVariables=" + bindVariables + "]"`